src/wp-includes/class-wp-theme-json-resolver.php

```diff
@@ -272,7 +321,7 @@ public static function get_theme_data( $deprecated = array(), $options = array()
 	public static function get_block_data() {
 		$registry = WP_Block_Type_Registry::get_instance();
 		$blocks   = $registry->get_all_registered();
-		$config   = array( 'version' => 1 );
+		$config   = array( 'version' => 2 );
```

Comment:
This is a bug I found while working in this PR. I can prepare a different PR if that's preferred but thought it was so minor it didn't merit one.

This sets the version of the `theme.json` schema used by the data stored in the blocks. We introduced version 2 in WordPress 5.9 and blocks only gained this functionality in this release, so we should be using the latest. In practice, because we're using the version 2 to store data in the blocks, probably we won't notice anything. Though this is still a dormant bug waiting to happen, and I rather do the right thing now. See [docs](https://developer.wordpress.org/block-editor/reference-guides/theme-json-reference/).